tests/redirects/map_external.py

```diff
@@ -9,5 +9,6 @@
 URLS = flatten((
     url_test('https://www.mozilla.com/', 'https://www.mozilla.org/firefox/'),
     url_test('https://aurora.mozilla.org/', 'https://www.mozilla.org/firefox/aurora/', status_code=requests.codes.found),
-    url_test('https://beta.mozilla.org/', 'https://www.mozilla.org/en-US/firefox/channel/#beta'),
+    # Bug 1299947 once new /channel pages are live we can update this redirect test.
+    # url_test('https://beta.mozilla.org/', 'https://www.mozilla.org/en-US/firefox/channel/#beta'),
```

Comment:
I'm not 100% sure if this test will still pass once the new pages are in production. The redirect should still end up in the right place, but we might need to update the expected final URL in the test here. I erred on the side that it will probably fail, but I could be happily mistaken.